draft-tuexen-opsawg-pcapng.xml

```diff
@@ -2037,6 +2043,204 @@ Section Header
+            identified by the same number (see <xref target="section_idb"/>)
+            of this field. The interface ID MUST be valid, which means that an
+            matching interface description block MUST exist.</t>
+
+            <t>Options: optionally, a list of options (formatted according to
+            the rules defined in <xref target="section_opt"/>) can be present.</t>
+
+          </list>
+        </t>
+
+        <t>Location information can be represented with in global (lat/lng/alt in degrees)
+        or local (X/Y/Z in metres) form, only one of these forms may be used in a given PCAP file.
+        Combining pcap files with different location formats should result in the 
+        removal of all position information.
+        </t>
+
+        <t>
+        Velocity and Orientation are to be interpreted from the zero frame for either form.
+        For the global form, X corresponds to latitude, Y to longitude, and Z to altitude.
+        Yaw is about the X/Y or Lat/Lng plane with zero facing along Y or North, 
+        pitch is applied with this yaw offset, and roll is applied to the combination of both.
+        </t>
+
+        <t>All the information fields are defined as options in order to support
+        systems that do not have a complete set of information. Therefore,
+        In addition to the options defined in <xref target="section_opt"/>,
+        the following options are valid within this block:</t>
+
+        <texttable anchor="options_cib" title="Capture Information Block Options">
+          <ttcol align="left">Name</ttcol>
+          <ttcol align="left">Code</ttcol>
+          <ttcol align="left">Length</ttcol>
+          <ttcol align="left">Multiple allowed?</ttcol>
+
+          <c>local_location</c>
+          <c>2</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>local_location_error</c>
+          <c>3</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location</c>
+          <c>4</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location_error</c>
+          <c>5</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation</c>
+          <c>6</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation_error</c>
+          <c>7</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity</c>
+          <c>8</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity_error</c>
+          <c>9</c>
+          <c>12</c>
+          <c>no</c>
+        </texttable>
+
+        <t>
+          <list hangIndent="8" style="hanging">
+
+            <t hangText="cib_local_location"><vspace blankLines="0"/>The 
+            cib_local_location option specifies the local location of the packet 
+            capture or interface in an abitrary plane; location is stored as three 
+            32-bit floats representing X, Y (in plane), and Z (altitude) offsets in \
```

Comment:
"in {backslash} metres"?  That looks incomplete; is there supposed to be something before or after the backslash, or is the backslash not supposed to be there, i.e. it should be "offsets in metres"?